src/styles/blocks/html.scss

```diff
@@ -0,0 +1,3 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Add also default `font-weight` and `font-size` for page here, so you will not need to repeat it in every class